test/verify/check-journal

```diff
@@ -185,13 +181,9 @@ ExecStart=/bin/sh -c 'for s in $(seq 10); do echo SLOW; sleep 0.1; done; sleep 1
 
         m.execute("systemctl start slow10")
 
-        if not "debian" in os.environ.get("TEST_OS", "") and "222-10" in m.execute("rpmquery systemd || true"):
-            # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1292805
-            # --follow is broken again, so we need to switch pages explicitly
-            # and hope that the service has started sending out messages in the meantime...
-            b.go("#/?prio=*&start=oldest&service=nonexisting.service")
-            wait_journal_empty()
-            b.go("#/?prio=*&start=oldest&service=slow10.service")
+        b.go("#/?prio=*&start=oldest&service=nonexisting.service")
```

Comment:
I would think that removing the HACK would mean removing these lines. You enabled the workaround for all tests.